presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix rare (but practical) incorrect query result for queries involving timestamp,
+  when the time zone database embedded in the version of Joda-Time that Presto depends on
+  does not match the time zone database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when optimize_mixed_distinct_aggregation is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add :func:`reduce_agg` aggregate function.
+* Add :func:`milliseconds` function (issue #9524).
+* Add support for predicate pushdown for partition symbols for window operators.
+* Remove SHOW STATS ON syntax
+* Restrict WHERE clause in SHOW STATS to filters that can be pushed down
```

Comment:
clarify that it's `pushed down to the connector`.  We sometimes use "pushed down" to mean pushed down within the plan and sometimes pushed down to the connector.

Reply:
Need a `.` at the end.